front-end/src/components/Content/Pagination/style.scss

```diff
@@ -0,0 +1,75 @@
+.pager{
+  padding-top:20px;
+  padding-left:360px;
+  padding-bottom:570px;
+}
+.pager  ul.pagination {
```

Comment:
@Taiga17 -san
If you already use scss. Please code follow SCSS  it will better.
Thanks